digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/dao/ExportReportDAO.scala

```diff
@@ -0,0 +1,95 @@
+package fi.liikennevirasto.digiroad2.dao
+
+import fi.liikennevirasto.digiroad2.asset.AssetTypeInfo
+import fi.liikennevirasto.digiroad2.oracle.MassQuery
+import fi.liikennevirasto.digiroad2.{ExportStatusInfo, Status}
+import org.joda.time.DateTime
+import org.slf4j.LoggerFactory
+import slick.driver.JdbcDriver.backend.Database.dynamicSession
+import slick.jdbc.{GetResult, PositionedResult}
+import slick.jdbc.StaticQuery.interpolation
+
+
+class ExportReportDAO {
+  val logger = LoggerFactory.getLogger(getClass)
+
+  lazy val municipalityDao = new MunicipalityDao
+
+  implicit val getResult = new GetResult[ExportStatusInfo] {
+    def apply(r: PositionedResult) : ExportStatusInfo = {
+      val id = r.nextLong()
+      val fileName = r.nextString()
+      val status = r.nextInt()
+      val createdDate = r.nextTimestampOption().map(timestamp => new DateTime(timestamp))
+      val createdBy = r.nextStringOption()
+      val exportedAssets = r.nextString()
+      val municipalities = r.nextString()
+      val content = r.nextStringOption()
+
+      val assetsName = if ( exportedAssets == null ) //can occur with status = 2
+                          ""
+                        else {
+                          exportedAssets.split(",")
+                                        .map( asset => AssetTypeInfo(asset.toInt).nameFI )
+                                        .mkString(",")
+                        }
+
+      val municipalitiesName = if ( municipalities == null ) //can occur with status = 2
+                                  ""
+                                else {
+                                  val municipalitiesAsset = municipalities.split(",").map(_.toInt).toSet
+                                  municipalityDao.getMunicipalitiesNameAndIdByCode(municipalitiesAsset)
+                                                .map(_.name)
+                                                .mkString(",")
+                                }
```

Comment:
If it can be null then you can us r.nextStringOption() in both exportedAssets and municipalities .

You then can do the .getOrElse("").split........